frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/DeleteBranchOnSlideOutSuggestionDialog.kt

```diff
@@ -0,0 +1,82 @@
+package com.virtuslab.gitmachete.frontend.actions.dialogs
+
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.ui.DialogWrapper
+import com.intellij.openapi.ui.Messages
+import com.intellij.ui.layout.panel
+import com.virtuslab.gitmachete.frontend.actions.base.BaseSlideOutBranchAction.DELETE_LOCAL_BRANCH_ON_SLIDE_OUT_GIT_CONFIG_KEY
+import com.virtuslab.gitmachete.frontend.resourcebundles.GitMacheteBundle.format
+import com.virtuslab.gitmachete.frontend.resourcebundles.GitMacheteBundle.getString
+import java.awt.event.KeyEvent
+import javax.swing.Action
+import javax.swing.JTextPane
+
+data class SlideOutOptions(
+    @get:JvmName("shouldRemember")
+    val remember: Boolean = false,
+    @get:JvmName("shouldDelete")
+    val delete: Boolean = false)
+
+class DeleteBranchOnSlideOutSuggestionDialog constructor(project: Project) :
+    DialogWrapper(project, /* canBeParent */ true) {
+
+  private var remember = false
+  private var delete = false
+
+  init {
+    title = getString("action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.title")
+    setResizable(false)
+    super.init()
+  }
+
+  fun showAndGetSlideOutOptions() = if (showAndGet()) SlideOutOptions(remember, delete) else null
+
+  override fun createCenterPanel() =
+      panel {
```

Comment:
Nit: I'm getting an unnaturally high vertical padding b/w the text and buttons 🤔 

![image](https://user-images.githubusercontent.com/3383210/103527475-b1c31800-4e82-11eb-95b1-d4a2ad47b802.png)